crates/storage/Cargo.toml

```diff
@@ -21,6 +21,7 @@ anyhow = { workspace = true }
 fuel-core-types = { workspace = true }
 fuel-vm-private = { workspace = true }
 mockall = { workspace = true, optional = true }
+rocksdb = { version = "0.19", default-features = false }
```

Comment:
Only `fuel-core` should depend on the `rocksdb`. It is better to do this mapping in the `fuel-core`

Reply:
Made a wrapper type and fixed it